features/bootstrap/SubjectContext.php

```diff
@@ -412,6 +414,63 @@ public function thereAreHighlightedArticlesWithTheMSA(int $number, string $subje
     }
 
     /**
+     * @Given /^there are (\d+) senior editors for the MSA \'([^\']*)\'$/
+     */
+    public function thereAreSeniorEditorsForTheMSA(int $number, string $subject)
+    {
+        $this->numberOfSeniorEditors = $number;
+
+        $seniorEditors = [];
+
+        $subjectId = $this->createSubjectId($subject);
+
+        $this->mockSubject($subject);
+
+        for ($i = $number; $i > 0; --$i) {
+            $seniorEditors[] = [
+                'id' => "$i",
+                'type' => [
+                    'id' => 'senior-editor',
+                    'label' => 'Senior editor',
+                ],
+                'name' => [
+                    'preferred' => "Person $i",
+                    'index' => "Person $i",
+                ],
+                'research' => [
+                    'expertises' => [
+                        [
+                            'id' => $subjectId,
+                            'name' => $subject,
+                        ],
+                    ],
+                    'focuses' => [],
+                ],
+            ];
+        }
+
+        foreach (array_chunk($seniorEditors, 100) as $i => $seniorEditorChunk) {
+            $page = $i + 1;
+
+            $this->mockApiResponse(
+                new Request(
+                    'GET',
+                    "http://api.elifesciences.org/people?page=$page&per-page=100&order=asc&subject[]=$subjectId&type=senior-editor",
```

Comment:
I don't think we will get to more than 100 senior editors for a single subject, which would create problems here